README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
 │
 ├── reference/                        <- Any reference material relevant to the project
 │
-├── src/                              <- Source data for the project 
+├── src/                              <- Source data,python scripts for the project 
 │
 ├── test/                             <- Files necessary for running model tests (see documentation below) 
 │
 ├── app.py                            <- Flask wrapper for running the model 
-├── run.py                            <- Simplifies the execution of one or more of the src scripts  
+├── create_db.py                      <- Creates the video game database schema and populates data into it.
+├── Dockerfile                        <- Docker file to build image video_games and run app
+├── run.py                            <- Simplifies the execution of one or more of the src scripts - stores data in s3 bucket
 ├── requirements.txt                  <- Python package dependencies 
+├── run_mysql_client.sh               <- Runs interactive sql server based on host,username,password.
 ```
 
 ## Running the app
-### 1. Initialize the database 
+### 1.Clone this repository-development
+Run the following command:
+
+    git clone https://github.com/aditya9729/2020-msia423-gudal-gamexplorer.git
+    
+Once cloned run to switch to the development branch:
+	
+	git checkout development
+
+### 2.Data Acquisition
```

Comment:
There is a way to download kaggle data programatically. They have recently released a CLI ([https://github.com/floydwch/kaggle-cli](linked here). this means you can carry out the data acquisition steps in your Dockerfile.

Reply:
Oh that sounds great. I will look into this. I was planning on shifting everything to a single dockerfile.